File: output/ChatGPT/codenet/C++/Java/s573429823.java
import java.util.Scanner;

public class s573429823 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long n = scanner.nextLong();
        boolean flg = false;

        if (n < 4) {
            System.out.println("No");
        } else {
            for (int i = 0; i <= (n / 4); i++) {
                for (int j = 0; j <= (n / 7); j++) {
                    if ((i * 4 + j * 7) == n) {
                        flg = true;
                        break;
                    }
                }
                if (flg) {
                    break;
                }
            }
            if (flg) {
                System.out.println("Yes");
            } else {
                System.out.println("No");
            }
        }
    }
}

File: output/ChatGPT/codenet/C++/Python/s573429823.py
n = int(input())

if n < 4:
    print("No")
else:
    flg = False
    for i in range(n // 4 + 1):
        for j in range(n // 7 + 1):
            if i * 4 + j * 7 == n:
                flg = True
                break
        if flg:
            break
    
    if flg:
        print("Yes")
    else:
        print("No")